Get-Started.md

```diff
@@ -1,44 +1,45 @@
 # Tutorial
-- This tutorial will guide you from the very basics to real example of IPLD.
-- IPLD stands for the Inter Planetary Linked Data
-- When we get data from the internet, we know it comes from the right place, if the URL is https (some times not even! TODO: add symantec & fake certificates), but we don’t know if it was the data we intended to get!
-- The same problem is with links, we don’t know if a link is still linking to the right page
-- Finally, the general problem of the current web is that we trust the source to give us the right data, so other computers in the network cannot really help sharing the same data!
-- These 3 properties are secure linking, distributed naming and immutable content!
-- Where does IPLD fit into this? IPLD is a data format to describe datasets where data can link to other data via cryptographic hashing, IPLD also provides a path scheme to point to some data in the datasets.
-- Let’s unroll what we just said.
+
+This tutorial will teach you about how to use IPLD. We'll start from the very basics, and end with some examples.
+
+IPLD stands for InterPlanetary Linked Data. This is based off of [IPFS](https://ipfs.io), which is the InterPlanetary File System. IPFS starts with a desire to have filesystems that work even when there are long delays involved (such as would be the case with interplanetary space travel). IPLD builds on top of this by allowing us to reference data in IPFS - or elsewhere.
```

Comment:
"interplanetary space travel!" :)
It is too exciting not to put a !
(your choice!)